Code/GraphMol/FileParsers/GeneralFileReader.h

```diff
@@ -104,7 +104,7 @@ std::unique_ptr<MolSupplier> getSupplier(const std::string& path,
 
   std::istream* strm;
   if (compressionFormat.empty()) {
-    strm = new std::ifstream(path.c_str());
+    strm = new std::ifstream(path.c_str(), std::ios::in | std::ios::binary);
```

Comment:
Does opening in binary mode have any other side effects?  I think this assumes the file is ASCII which I think is a fair assumption except when reading UTF-8 csv files.

Reply:
@bp-kelley It won't affect the encoding - the low-level C functions will read bytes in both cases.
`rb` is identical to `r` for POSIX systems:
https://www.gnu.org/software/libc/manual/html_node/Opening-Streams.html
It only affects non-POSIX systems like Windows, and its scope is only limited to how newlines are handled:
http://c-faq.com/stdio/textvsbinary.html
As for why MinGW `ftell` does strange things on files opened in text mode, I found this:
https://lists.gnu.org/archive/html/bug-gnulib/2008-03/msg00256.html
As the offsets are only used to jump to the right position in the file with `fseek` and newlines are stripped anyway by the tokenizer, I think this is safe.